db/migrate/20211030114756_add_user_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToPosts < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] According to the ERD scheme, you should refer to the users as authors for posts, likes, and comments. Please update these relationships. You can take my suggestion as an example. Please check the ERD schema and update your models accordingly. You can take my change suggestion as an example. 

```suggestion
add_reference :posts, :author, index: true
add_foreign_key :posts, :users, column: :author_id
```